lib/type.ml

```diff
@@ -223,3 +223,34 @@ let rec of_absform = function
      of_elem (TySingleton (Atom "not_implemented"))
   | F.TyPredef {line; name; args} ->
      failwith (!%"Prease report: line:%d: unexpected predef type: '%s/%d'" line name (List.length args))
+
+let rec of_erl_type = function
+  | Erl_type.Any -> TyAny
+  | None -> TyBottom
+  | Unit -> TyBottom
+  | erl_type -> TyUnion (union_of_erl_type erl_type)
+and union_of_erl_type = function
+  | Erl_type.None -> []
+  | AnyAtom -> [TyAtom]
+  | AtomUnion atoms -> List.map ~f:(fun atom -> TySingleton (Constant.Atom atom)) atoms
+  | Function {params; ret} -> [TyFun (params |> List.map ~f:of_erl_type, of_erl_type ret)]
+  | Number _ -> [TyNumber]
```

Comment:
It's not correct. `of_erl_type (Number IntSet [1])` should be `of_elem (TySingleton (Number 1))`.

Reply:
I fixed to correctly handle number types.

Reply:
The case of `IntSet` with multiple integer does not work yet. If it is a future task, will you add a comment like:
```ocaml
(* TODO ... *)
```

Reply:
Fixed with 573871f